SwiftScripts/Package.resolved

```diff
@@ -15,43 +15,34 @@
         "repositoryURL": "https://github.com/stephencelis/SQLite.swift.git",
         "state": {
           "branch": null,
-          "revision": "7a2e3cd27de56f6d396e84f63beefd0267b55ccb",
-          "version": "0.14.1"
+          "revision": "4d543d811ee644fa4cc4bfa0be996b4dd6ba0f54",
+          "version": "0.13.3"
```

Comment:
What's the reason we're going backwards in our dependency versions here?

Reply:
No clue. I'll run an package update?